### PR TITLE
Room procedures

### PR DESCRIPTION
**Pull request by La7rodectus**

### CODE REVIEW

**Review by lizardlynx — APPROVED** [review]

**Review by mezgoodle — APPROVED** [review]

File: Database/Procedures/2020_12_14_006_SP_roomUpdate.sql [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+-- =============================================
+-- Author:		<Author,,Name>
+-- Create date: <Create Date,,>
+-- Description:	<Description,,>
+-- =============================================
+CREATE PROCEDURE [dbo].[UpdateRooms]
+@RoomID int,
+@Price int,
+@RoomNumber int,
+@TV int,
```
Comment (VsIG-official) [inline Database/Procedures/2020_12_14_006_SP_roomUpdate.sql:17]: is TV should be "int"? I thougth, that it should be bit or bool

File: Database/Procedures/2020_12_14_007_SP_SetRoom.sql [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+GO
+-- =============================================
+-- Author:		<Author,,Name>
+-- Create date: <Create Date,,>
+-- Description:	<Description,,>
+-- =============================================
+CREATE PROCEDURE [dbo].[SetRoom] 
+@Price int,
+@RoomNumber int,
+@TV int,
```
Comment (VsIG-official) [inline Database/Procedures/2020_12_14_007_SP_SetRoom.sql:16]: Same question from above

**Review by VsIG-official — COMMENTED** [review]
Is it bool, int or bit?

File: Database/Procedures/2020_12_14_007_SP_SetRoom.sql (hunk 2 above)
Comment (La7rodectus) [reply Database/Procedures/2020_12_14_007_SP_SetRoom.sql:16]: Agree with u, TV have to be bit type. Will change.

**Review by VsIG-official — APPROVED** [review]
Seems good

**Pull request merged**